### PR TITLE
admin/update-from-cc-for-dependabot-fix

### PR DESCRIPTION
**Pull request by evamaxfield**
The same PR as the missoula one. Updates to cookiecutter to resolve the dependabot stuff.

Merge whenever and the current dependabot PRs should go away / be closeable regardless.

### CODE REVIEW

File: infra/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+cdp-backend==3.2.3
```
Comment (whargrove) [inline infra/requirements.txt:1]:
```suggestion
cdp-backend==3.2.11
```

File: python/setup.py [hunk 2]
```diff
@@ -6,18 +6,13 @@
 from setuptools import find_packages, setup
 
 requirements = [
-    "cdp-backend==3.2.0",
-]
-
-pipeline_requirements = [
-    "cdp-backend[pipeline]==3.2.0",
+    "cdp-backend[pipeline]==3.2.3",
```
Comment (whargrove) [inline python/setup.py:9]:
```suggestion
    "cdp-backend[pipeline]==3.2.11",
```

**Review by whargrove — CHANGES_REQUESTED** [review]
@evamaxfield Sorry it has taken a while to review this and thanks of course for the fix. IIUC the merge conflicts are caused by subsequent version bumps of cdp-backend we've applied, I've suggested some fixes here so that it can be merged.

Are you able to take a pass and make sure the merges resolve cleanly with the version of cdp-backned we need currently?

File: cookiecutter.yaml [hunk 3]
```diff
@@ -11,3 +11,5 @@ default_context:
     hosting_github_url: "https://github.com/OpenMontana/montana-legislature-council-data-project"
     hosting_web_app_address: "https://OpenMontana.github.io/montana-legislature-council-data-project"
     firestore_region: "us-west1"
+    event_gather_timedelta_lookback_days: 2
```
Comment (evamaxfield) [inline cookiecutter.yaml:14]:
```suggestion
    event_gather_timedelta_lookback_days: 10
```

@whargrove for your tracking. smai had updated this in the app prior to the base app actually tracking it in the default build: https://github.com/OpenMontana/montana-legislature-council-data-project/blob/main/python/event-gather-config.json#L7

**Review by evamaxfield — COMMENTED** [review]
I think this is good again. Feel free to merge. Worst case is that you just need to rerun the cookiecutter

**Review by whargrove — APPROVED** [review]

**Pull request merged**